iosApp/iosApp/Pages/AlertDetails/AlertDetailsPage.swift

```diff
@@ -62,7 +62,7 @@ struct AlertDetailsPage: View {
                     .scaledToFit()
                     .frame(maxHeight: modeIconHeight, alignment: .topLeading)
             } else if alert?.effect == .elevatorClosure {
-                AlertIcon(alertState: .elevator)
+                AlertIcon(alertState: .elevator, accessibilityHidden: true)
```

Comment:
The QA also mentioned potentially moving the elevator alert icon next to the alert header. This was giving me more trouble than expected on iOS w/ layout issues (below is just putting the two together in an hstack), so I opted to leave it out of this, but open to suggestions if there seems like an easy fix! I also tried using the `.fixedSize(horizontal: false, vertical: true)` modifier on the hstack with no luck.
![IMG_0286](https://github.com/user-attachments/assets/b2b9028a-5da3-4d32-83f8-d000af8499e6)

Reply:
This definitely should be easy, but if SwiftUI is doing SwiftUI things and none of the things that should work actually work, it makes sense to not sink infinite time into this in this PR.